taichi/analysis/same_statements.cpp

```diff
@@ -196,6 +196,24 @@ class IRNodeComparator : public IRVisitor {
     basic_check(stmt);
   }
 
+  void visit(MatrixInitStmt *stmt) override {
```

Comment:
Where is this used?

Reply:
iirc I was trying to add CSE support for matrix

Reply:
Got it. Let's make each PR clean :-)